.vscode/launch.json

```diff
@@ -44,7 +43,23 @@
         "--inspect-brk",
         "${workspaceRoot}/node_modules/.bin/jest",
         "--runInBand",
-        "--coverage",
+        "false",
+        "${relativeFile}"
+      ],
+      "console": "integratedTerminal"
+  },
+  {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug current E2E test file",
+      "skipFiles": [
+          "<node_internals>/**"
+      ],
+      "runtimeArgs": [
+        "--inspect-brk",
+        "${workspaceRoot}/node_modules/.bin/jest",
+        "--runInBand",
+        "--config=test/jest-ci.json",
```

Comment:
What is the context around this change?

Reply:
It's just a launch configuration for Visual Studio Code. We have another one to run a test file with an attached debugger, but it doesn't work with E2E since the jest configuration is different, so I needed to pass `jest-ci.json` configuration to this one. I find this really useful to debug a complete request flow in an E2E test but I can keep it local if you prefer 🙂

Reply:
I was just trying to understand the context of this change because it might have been pushed by mistake.

In general I find it preferable to have this in a separate PR so that we have more context in the future why this was introduced. The context of this changes are around `/safe-apps` (given the PR title + description).

Reply:
I totally agree, my bad. I'll move this change to another PR 👍🏻

Reply:
Removed in [21ff50f](https://github.com/5afe/safe-client-gateway-nest/commit/21ff50f7bc77e3cb11bcbaa2c1cf0322c60cd08c)